Assets/Scripts/Runtime/ScriptableObjects/Stages.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Assets.Scripts.Runtime.Enums;
+using UnityEngine;
+
+namespace Assets.Scripts.Runtime.ScriptableObjects
+{
+    [CreateAssetMenu(fileName = "Stage", menuName = "Stages/new stage", order = 1)]
+    public class Stages : ScriptableObject
```

Comment:
This is a single stage description, why is the class called plural `stages` not `stage`